EIPS/eip-6963.md

```diff
@@ -75,6 +75,16 @@ data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12
 
 The `icon` string MUST be a data URI as defined in [RFC-2397]. The image SHOULD be squared with 96x96px minimum resolution. The image format is RECOMMENDED to be either lossless or vector based such as PNG, WebP or SVG to make the image easy to render on the DApp. Since SVG images can execute Javascript, applications and libraries MUST render SVG images using the `<img>` tag to ensure no untrusted Javascript execution can occur.
 
+#### RDNS
+
+The **`rdns`** (Reverse-DNS) property serves to provide an identifier which DApps can rely on to be stable between sessions. The Reverse Domain Name Notation is chosen to prevent namespace collisions.
+The Reverse-DNS convention implies that the value should start with a reversed DNS domain name controlled by the Provider. The domain name should be followed by a subdomain or a product name. Example: `com.example.MyBrowserWallet`.
+
+- The DNS part of the `rdns` value MUST BE a valid [RFC-1034] Domain Name;
+- The DNS part of the `rdns` value SHOULD BE an active domain controlled by the Provider;
```

Comment:
Likewise
```suggestion
- The root domain of the `rdns` value SHOULD BE an active domain controlled by the Provider;
```
Can't be a must because we don't provide a mechanism for making this testable, but hints that a future extension spec might.

Reply:
I don't think we've defined a "_root domain_" thought 🤔

Reply:
```suggestion
- The DNS part of the `rdns` value SHOULD BE an active apex domain controlled by the Provider;
```
Googling around, I realized "root" isn't really a stable or defined term.  This recent RFC supersedes the older ones and defines `apex` as the "highest delegation point owned by someone", basically: 
https://www.rfc-editor.org/rfc/rfc8499